rna_transcription_updated.rb

```diff
@@ -0,0 +1,26 @@
+class Complement
+  def self.of_dna(dna)
+    dna=dna.split('');
+    rna=''
+    (0..dna.length-1).each do |i|
```

Comment:
add brackets before range ends. (dna.legth-1). can we use `...` instead of `..` in this case?